cvat/apps/engine/backup.py

```diff
@@ -542,7 +559,7 @@ def _export_task(self, zip_obj, target_dir=None):
         self._write_annotations(zip_obj, target_dir)
         self._write_annotation_guide(zip_obj, target_dir)
 
-    def export_to(self, file, target_dir=None):
+    def export_to(self, file: str | ZipFile, target_dir: str | None = None):
```

Comment:
Not an error, but such annotations in inherited functions tend to become outdated quickly. Typically it's better to just use annotations from the original function instead of duplicating them everywhere.